cmd/bnscli/cmd_key.go

```diff
@@ -83,6 +101,38 @@ Print out a hex-address associated with your private key.
 			Ed25519: raw,
 		},
 	}
-	_, err = fmt.Fprintln(output, key.PublicKey().Address())
+
+	bech, err := bech32.Encode("iov", key.PublicKey().GetEd25519())
```

Comment:
The data input must be the 20 bytes weave address, not the pubkey. See this diff for clarification: https://github.com/iov-one/iov-core/pull/1158/files

Reply:
Fixed and tested :muscle: